src/data/quot.lean

```diff
@@ -200,6 +200,21 @@ protected def lift_on₂' (q₁ : quotient s₁) (q₂ : quotient s₂) (f : α
   (h : ∀ a₁ a₂ b₁ b₂, @setoid.r α s₁ a₁ b₁ → @setoid.r β s₂ a₂ b₂ → f a₁ a₂ = f b₁ b₂) : γ :=
 quotient.lift_on₂ q₁ q₂ f h
 
+-- I (Buzzard) am reluctant to write "@[elab_as_eliminator, reducible]", not because I know
+-- it's wrong, but because I don't really understand these elaborator tags
+-- and don't want to write stuff which makes it look like I know what I'm doing;
+-- also I note some of these functions below are [reducible] and others aren't.
+-- Is `protected` correct?
```

Comment:
They should be `elab_as_eliminator` but not `elab_strategy` or `reducible`. They're theorems so there's no point making them reducible

Reply:
They're theorems now, at least :-) I have tagged them correctly now.